pkg/epinio/utils/git.ts

```diff
@@ -0,0 +1,119 @@
+import { EPINIO_APP_ENV_VAR_GIT } from '../types';
+import startCase from 'lodash/startCase';
```

Comment:
Think we're trying t o back out of using lodash. There's a file with some basic string utils `/shell/utils/string.js`. Would `ucFirst` work instead?